client/src/components/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Button, Theme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+import { HeaderProps } from '../types/initialTypes';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    padding: theme.spacing(3, 5),
+
+    '& .MuiButton-text': {
+      textTransform: 'capitalize',
+      color: theme.palette.secondary.main,
+    },
+  },
+}));
+
+export const Header: React.FC<HeaderProps> = ({ handleShowPopup }) => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <span>MyBlog</span>
+      <div>
+        <Button variant='text'>Home</Button>
+        <Button variant='text' onClick={handleShowPopup}>New Article</Button>
+      </div>
+    </div>
```

Comment:
you can use `header` for the container, and `nav` for the buttons container
instead of div's. It will be more semantically correct.

Reply:
Yes, I agree. Thanks)